### PR TITLE
Added notes in project

### LINKED ISSUE #129: Add notes to a project
When I create a project I would like to add some descriptive notes stored with the project and I could not find a way to do that

### PR DESCRIPTION
**Pull request by DhruviJ17**
Add notes attribute for the projects
Fixes #129

Signed-off-by: Dhruvi Jodhawat dhruvi.jodhawat@gmail.com

### CODE REVIEW

File: scanpipe/forms.py [hunk 1]
```diff
@@ -107,19 +107,26 @@ class Meta:
...
         ]
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
         name_field = self.fields["name"]
+        notes_field = self.fields["notes"]
         name_field.widget.attrs["class"] = "input"
         name_field.widget.attrs["autofocus"] = True
         name_field.help_text = "The unique name of your project."
+        notes_field.help_text = "Add notes for your project."
```
Comment (tdruez) [inline scanpipe/forms.py:120]: Why are you overriding the help_text here?

**Review by tdruez — COMMENTED** [review]
Hi @DhruviJ17 some feedback on your PR:
- The textarea field rendering is not consistent with the rest of the form
Screenshot 2021-09-14 at 5 02 22 PM
- Notes value rendering in the project details view should not be displayed if empty:
Screenshot 2021-09-14 at 5 02 09 PM
- Also, it's quite unreadable when filed:
Screenshot 2021-09-14 at 5 03 08 PM

- Missing entry in the CHANGELOG
- Missing unit test


I think the notes should be something that can be added later on a Project but may not need to be part of the creation process.

@pombredanne could you precise your need for such a field?

**Comment by DhruviJ17** [comment]
Should I go ahead with the changes then or should work on this in future? @tdruez

**Comment by pombredanne** [comment]
@DhruviJ17 I think this was closed by mistake.

**Comment by pombredanne** [comment]
@DhruviJ17 closing this for now for lack of activity. Please re-open if you want to complete this! Thanks!

**Pull request closed without merging**